modules/openapi-generator-cli/src/main/java/org/openapitools/codegen/cmd/Generate.java

```diff
@@ -199,8 +202,23 @@
             description = "Skips the default behavior of validating an input specification.")
     private Boolean skipValidateSpec;
 
+    @Option(name = {"--log-to-stderr"},
```

Comment:
I think this option is much clearer, and has the benefit of not being a breaking change.